utils.go

```diff
@@ -66,3 +81,55 @@ func IsBinary(data []byte) bool {
 
 	return true
 }
+
+func loadGitattributes() (map[string]string, error) {
+	gitAttributes := map[string]string{}
+	data, err := ioutil.ReadFile(".gitattributes")
+	if err != nil {
+		if err != os.ErrNotExist {
+			log.Println(".gitattributes: " + err.Error())
+		}
+		return nil, err
+	}
+
+	if data != nil {
+		tokens := strings.Fields(string(data))
+		for i := 0; i < len(tokens); i = i + 2 {
+			gitAttributes[tokens[i]] = tokens[i+1]
+		}
+	}
+
+	return gitAttributes, nil
+}
+
+func parseAttributes(attributes map[string]string) {
+	for key, val := range attributes {
+		switch {
+		case val == "enry-vendored" || val == "enry-documentation":
+			gitAttributes[key] = true
+		case val == "enry-vendored=false" || val == "enry-documentation=false":
+			gitAttributes[key] = false
+		case strings.Contains(val, "enry-language="):
+			processLanguageAttr(key, val)
+		default:
+			log.Printf("The matcher %s doesn't exists", val)
+		}
+	}
+}
+
+func processLanguageAttr(regExpString string, attribute string) {
+	tokens := strings.Split(attribute, "=")
+	if len(tokens) == 2 {
+		regExp, err := regexp.Compile(regExpString)
+		if err == nil {
+			languageGitAttributes[regExp] = tokens[1]
```

Comment:
you should pass tokens[1] to GetLanguageByAlias() before store it in the map, this ensures the language will be stored like a "standard name" if it is a known language for enry